packages/sync/src/links/LinksBuilder.ts

```diff
@@ -20,12 +20,8 @@ import { LocalDirectiveFilterLink } from "./LocalDirectiveFilterLink";
 export const defaultHttpLinks = async (config: DataSyncConfig): Promise<ApolloLink[]> => {
   const offlineQueueLink = new OfflineQueueLink(config, "mutation");
   const localDirectiveFilterLink = new LocalDirectiveFilterLink();
-
   let httpLink = new HttpLink({ uri: config.httpUrl, includeExtensions: config.auditLogging }) as ApolloLink;
-  if (config.headerProvider) {
-    httpLink = concat(createHeadersLink(config), httpLink);
-  }
-
+  httpLink = concat(createHeadersLink(config), httpLink);
```

Comment:
This affects integration tests. Like previous version more ;)
"There is no link if there is no headers you know"